docs/HACKING.md

```diff
@@ -157,3 +157,38 @@ the operator since the MCD daemonset will be overwritten:
 ```
 oc scale deployment machine-config-operator --replicas=0
 ```
+
+# The test suites
+
+We have a few contexts that run on pull requests. `unit` runs `make test-unit`.
+The `e2e-aws` job is common across most OpenShift repos; it runs the installer
+with a custom update payload using code from the PR, and then runs the e2e test
+suite from [OpenShift Origin](https://github.com/openshift/origin/).
+
+Finally, [recently we added](https://github.com/openshift/release/pull/2577) an
+`e2e-aws-op` job. This one also generates a cluster with the code, but runs
+`make test-e2e` from our own repo rather than Origin's test suite. We can run
+destructive tests here.
+
+## Debugging a test suite
+
+In the `pods/` directory in "artifacts" you can find the logs for the various
+components.  Also you can see the node annotations in `nodes.json`; usually
+the `machineconfigs.json` and `machineconfigpools.json` are also there too.
+
+## Logging into a cluster live
+
+Login: `oc login https://api.ci.openshift.org/console/`
+
+In the pull request, you'll see a "project" or Kubernetes namespace, so you can e.g.: `oc project ci-op-zgctgrpy`
```

Comment:
Where is this in the PR? I've never seen it? Can we add a picture or something?

Reply:
Updated :arrow_down:

Reply:
so colin helped me find this, but it's quite hidden so i think we can be explicity about where it is located, ie click on the aws e2e test - click on raw build log - show some example lines where it is.

Reply:
perf!